src/main/java/com/synopsys/integration/jenkins/detect/service/DetectArgumentService.java

```diff
@@ -101,7 +101,7 @@ private boolean validateExpandedArguments(String argument) {
     private String escapeArgument(String argument, Function<String, String> escaper) {
         // Assume a cleaned argument, then test
         String cleanedArg = argument;
-        if (argument.startsWith("--") && argument.contains("=")) {
+        if (argument.startsWith("--") && argument.contains("=") && !argument.contains("&")) {
```

Comment:
Do you know if we have a similar escaping logic in Detect? Looks fine to me as long as we know there are no other detect properties where escaping '&' is required. I couldn't find an example, but for sanity I'd look at what Detect core is doing

Reply:
I looked at Detect Core to see how escaping is done but couldn't find a particular example where escaping '&' is required.